shiva/src/main/scala/nl/lumc/sasc/biopet/pipelines/shiva/ShivaReport.scala

```diff
@@ -105,13 +106,14 @@ trait ShivaReportTrait extends MultisampleMappingReportTrait {
   }
 
   /** Files page, can be used general or at sample level */
-  override def filesPage(sampleId: Option[Int] = None, libraryId: Option[Int] = None): ReportPage = {
-    val vcfFilesSection = if (variantcallingExecuted) List("VCF files" -> ReportSection("/nl/lumc/sasc/biopet/pipelines/shiva/outputVcfFiles.ssp",
-      Map("sampleId" -> sampleId)))
-    else Nil
-    val oldPage = super.filesPage(sampleId, libraryId)
-    oldPage.copy(sections = oldPage.sections ++ vcfFilesSection)
-  }
+  //TODO: fix files pages
```

Comment:
ah yes ty, this part was generalized but forgot to remove this from Shiva. I did remove this